zoneminder-latest.json

```
@@ -67,14 +67,14 @@
                 }
             }
         },
-        "description": "ZoneMinder: Free, open-source software to control IP, USB and Analog (CCTV) cameras (v1.32) - please note this runs as privileged in docker (to set shm to a higher amount).<p>Based on the following docker image: <a href='https://github.com/dlandon/zoneminder' target='_blank'>https://github.com/dlandon/zoneminder</a></p>.",
+        "description": "ZoneMinder: Free, open-source software to control IP, USB and Analog (CCTV) cameras (v1.34) - please note this runs as privileged in docker (to set shm to a higher amount).<p>Based on a custom docker image: <a href='https://hub.docker.com/r/dlandon/zoneminder' target='_blank'>https://hub.docker.com/r/dlandon/zoneminder</a>, available for amd64 and arm64 architecture.</p>",
```

```suggestion
        "description": "ZoneMinder: Free, open-source software to control IP, USB and Analog (CCTV) cameras (v1.34) - please note this runs as privileged in docker (to set shm to a higher amount).<p>Based on a custom docker image: <a href='https://hub.docker.com/r/dlandon/zoneminder' target='_blank'>https://hub.docker.com/r/dlandon/zoneminder</a>, available for amd64 architecture only.</p>",
```
@Hooverdan96 so it looks like we took a step back on this one!